virtualbox/__init__.py

```diff
@@ -24,16 +24,16 @@
 from multiprocessing import current_process
 
 from virtualbox.library_ext import library
-from .__about__ import (
-    __title__,  # noqa: F401
-    __version__,
-    __author__,
-    __author_email__,
-    __maintainer__,
-    __maintainer_email__,
-    __license__,
-    __url__,
-)
+# from .__about__ import (
```

Comment:
Commenting this out isn't a lint error fix?

Reply:
I have unmommented and fixed  errors